build.cabal.sh

```diff
@@ -61,6 +61,11 @@ else
     # Initialize sandbox if necessary
     if ! ( "$CABAL" sandbox hc-pkg list > /dev/null 2>&1); then
         "$CABAL" sandbox init
+        ( cd ../libraries/Cabal/Cabal
```

Comment:
I think you just need to replace this section with 

`cabal sandbox add-source ../libraries/Cabal/Cabal`